### DISCUSSION TITLE
実装迷わないための仮デザイン（すみません、勝手に作りました）

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#46] [【運営】相談・報告]
- デザインはある程度自由とのことだったのですが
- やっぱり悩みそうなので認識のすり合わせ含め作ってみました
- 素材: https://www.openpeeps.com/

---

- 日本っぽくとりあえず赤色採用
- とりあえずシンプルにしてます（後にわちゃわちゃにすれば良いかと）

![スクリーンショット 2024-03-16 2 04 01](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/bad6cb8f-cae9-4dea-9691-ad89cec3c63d)

**Comment by pss-aileen** [comment]
### サイトマップ
- TOPページ: ヒーローエリア程度
- ABOUT: README.md 貼り付け
- 個別ページ: ユーザーの情報を表示（実装はTOPができた後）

**Comment by kazzyfrog** [comment]
めっちゃ良いですね！！
さすがです！🤩

**Comment by kagomen** [comment]
わー！！！すごく良いですね！
シンプルポップで親しみ感もあり、素敵です……！🌟
ありがとうございます！！！
